projects/fsastorefrontlib/src/cms-components/sync-pilot/sync-pilot-connection.component.html

```diff
@@ -1,12 +1,16 @@
 <ng-container *ngIf="component$ | async as component">
-  <div class="row">
-    <button
-      class="primary-button col-12 col-md-3 m-auto btn-block"
-      (click)="
-        establishConnection(component.url, component.channel, component.action)
-      "
-    >
-      {{ component.linkName }}
-    </button>
+  <h3 class="table-header-title mb-0">
+    {{ 'fscommon.syncPilot.needHelp' | cxTranslate }}
+  </h3>
+  <div
+    class="notice link"
+    (click)="
+      establishConnection(component.url, component.channel, component.action)
+    "
+  >
+    <h4 class="fas fa-headset table-header-value notice"></h4>
```

Comment:
Should't we use `span` tag for icons?

Reply:
Since this is an empty tag I chose to follow the structure of the rest of the table. Putting `icon` in span here would require additional styling to get it in line with other headings. Also don't see the need to create an extra DOM node by putting `span` in `h4`.

Reply:
Didn't mean to add extra node, just to change `h4` -> `span,` but you explained reason for choosing `h4`.

Reply:
Yup, I got what you meant, but the other solution would be to put `span` inside `h4`, so I just wanted to explain why I've ruled that out too.